rpc/api/util.go

```diff
@@ -25,6 +29,19 @@ func (u *UtilApi) Encrypt(raw string, passphrase string) (string, error) {
 }
 
 func (u *UtilApi) RawToBalance(balance types.Balance, unit string, tokenName *string) (types.Balance, error) {
+	if tokenName != nil {
+		token, err := u.ledger.GetTokenByName(*tokenName)
+		if err != nil {
+			return types.ZeroBalance, err
+		}
+		if token.TokenId != common.ChainToken() {
+			b, err := balance.Div(int64(math.Pow10(int(token.Decimals))))
```

Comment:
should check `math.Pow10` result, it's could be zero.

Reply:
token.Decimals is nonnegative, so zero should not happen.  and if zero happen ,Div will return error , no panic

Reply:
when `token.Decimals` is big than 308, the result is not a number.

Reply:
forget about it, `Decimals`'s type is uint8...